lib/resources/r4/document_reference.yaml

```
@@ -29,3 +95,143 @@ fields:
     - display: US Core DiagnosticReport Category
       system: http://loinc.org
       info_link: http://hl7.org/fhir/us/core/ValueSet/us-core-diagnosticreport-category
+
+- name: subject
+  required: 'Yes'
+  type: Reference
+  action:
+    - create
+  description: Who or what the document is about.
+  note: Only patient references are supported.
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.subject
+  example: |
+    {
+      "subject": {
+        "reference": "Patient/1415936"
+      }
+    }
+
+- name: author
+  required: 'Yes'
+  type: Reference
+  action:
+    - create
+  description: Identifies who is responsible for adding the information to the document.
+  note: The author field is optional for system access.  For provider access, only a single author can be provided, and the author must be the authorized provider. Must contain only Practitioner references with valid millennium ids.
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.author
+  example: |
+    {
+      "author": [
+        {
+          "reference": "Practitioner/123456"
+        }
+      ]
+    }
+
+- name: authenticator
+  required: 'No'
+  type: Reference
+  action:
+    - create
+  description: Which person or organization authenticates that this document is valid.
+  note: The authenticator field is only supported with system access. This field cannot be populated for provider access. If set, must be a Practitioner reference with valid millennium id.
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.authenticator
+  example: |
+    {
+      "authenticator": {
+        "reference": "Practitioner/123456"
+      }
+    }
+
+- name: content
+  required: 'Yes'
+  type: BackboneElement
+  action:
+    - create
+  description: The document and format referenced. There may be multiple content element repetitions, each with a different format.
```

```suggestion
  description: The document and format referenced.
```

We can remove the references to multiple elements in the description since we don't support them.

---

You can find the changes here: https://github.com/cerner/fhir.cerner.com/pull/446/commits/eb5ca8d7a6905720c70e0c203d9de99739180394